src/Compilers/Shared/NativeMethods.cs

```diff
@@ -93,5 +95,81 @@ out PROCESS_INFORMATION lpProcessInformation
 
         [DllImport("kernel32.dll", CharSet = CharSet.Unicode, SetLastError = true)]
         internal static extern IntPtr GetCommandLine();
+
+#if !NET
+        //------------------------------------------------------------------------------
+        // ResolveLinkTarget
+        //------------------------------------------------------------------------------
+        extension(File)
+        {
+            public static FileSystemInfo? ResolveLinkTarget(string path, bool returnFinalTarget)
+            {
+                if (!returnFinalTarget) throw new NotSupportedException();
+
+                path = Path.GetFullPath(path);
```

Comment:
Is there a need for this method to work with relative paths?

Reply:
There is probably no need by the current code, but this polyfill could be used in the future by others so it seems it should behave just like the BCL's equivalent.

But it seems CreateFileW would handle relative paths anyway so the `GetFullPath` call seems unnecessary, I will remove it.